README.md

```diff
@@ -17,9 +17,50 @@ The path to dynawo deploy is then the path to the subdirectory `dynawo` in the d
 PATH_TO_DYNAWO_DEPLOY=${DYNAWO_HOME}/deploy/gcc8/shared/dynawo/
 ```
 
-Then generate the cmake cache in your build directory:
+### By script
 
-`cmake <SRC_DIR> -DCMAKE_BUILD_TYPE=Release -DDYNAWO_HOME=<DYNAWO_HOME> -DBOOST_ROOT=<DYNAWO_HOME> -DCMAKE_INSTALL_PREFIX=<INSTALL_DIR>`
+To build DynaFlow Launcher you need to clone the repository and launch the following commands in the source code directory, it will create a `myEnvDFL.sh` file that will be your personal entrypoint to launch DFL and parametrise some options.
+
+``` bash
+$> git clone https://github.com/dynawo/dynaflow-launcher
+$> cd dynaflow-launcher
+$> echo '#!/bin/bash
+
+# Required
+export DYNAFLOW_LAUNCHER_HOME=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+export DYNAWO_HOME=<PATH_TO_DYNAWO_DEPLOY>
+export DYNAFLOW_LAUNCHER_BUILD_TYPE=Debug
+export DYNAFLOW_LAUNCHER_LOCALE=en_GB
+
+# Optional
+# export DYNAFLOW_LAUNCHER_CMAKE_GENERATOR=Ninja # default is Unix Makefiles
+# export DYNAFLOW_LAUNCHER_PROCESSORS_USED=8 # default 1
+# export DYNAFLOW_LAUNCHER_SHARED_LIB=ON # default OFF
+# export DYNAFLOW_LAUNCHER_USE_DOXYGEN=OFF # default ON
+# export DYNAFLOW_LAUNCHER_BUILD_TESTS=OFF # default ON
+# export DYNAFLOW_LAUNCHER_LOG_LEVEL=INFO # default INFO: can be DEBUG, INFO, WARN, ERROR
+
+# Run
+$DYNAFLOW_LAUNCHER_HOME/scripts/envDFL.sh $@' > myEnvDFL.sh
+$> chmod +x myEnvDFL.sh
+```
+
+Then update the path "PATH_TO_DYNAWO_DEPLOY" in the file to your deployed installation of Dyna&omega;o and launch the following command:
+``` bash
+$> ./myEnvDFL.sh build-user
+```
+
+All commands described in the rest of this README are accessible throught this script. To access all options of the script, type:
+``` bash
+$> ./myEnvDFL.sh help
+```
+
+### Direct CMake compilation
+You can generate the cmake cache in your build directory without using the script. Go to your build directory and then launch:
+
+```bash
+$> cmake <SRC_DIR> -DCMAKE_BUILD_TYPE=Release -DDYNAWO_HOME=<DYNAWO_HOME> -DBOOST_ROOT=<DYNAWO_HOME> -DCMAKE_INSTALL_PREFIX=<INSTALL_DIR>
```

Comment:
One of the the point of having a utility script is to avoid maintaining this command in the readme

Reply:
OK